oauth_dropins/reddit.py

```diff
@@ -135,19 +135,17 @@ def praw_to_user(user):
     user: :class:`praw.models.Redditor`
 
   Note 1: accessing redditor attributes lazily calls reddit API
-  Note 2: subreddit (optional) refers to a user profile (stored as a subreddit)
+  Note 2: if user.is_suspended is True, other attributes will not exist
+  Note 3: subreddit refers to a user profile (stored as a subreddit)
```

Comment:
minor: mind including a https://praw.readthedocs.io/en/latest/code_overview/models/redditor.html link here for more info on these notes?

Reply:
actually nm, thought i had a bigger request, but nope, PR looks good. i'll merge and we can add this later.